apps/api/v2/package.json

```diff
@@ -12,13 +12,15 @@
     "dev:build:watch": "yarn workspace @calcom/platform-constants build:watch & yarn workspace @calcom/platform-utils build:watch & yarn workspace @calcom/platform-types build:watch",
     "dev:build": "yarn workspace @calcom/platform-constants build && yarn workspace @calcom/platform-enums build && yarn workspace @calcom/platform-utils build && yarn workspace @calcom/platform-types build ",
     "dev": "yarn dev:build && docker-compose up -d && yarn copy-swagger-module && yarn start --watch",
+    "dev:no-docker": "yarn dev:build && yarn copy-swagger-module && yarn start --watch",
```

Comment:
Added the option to run v2 with no docker. Happy to remove was just annoying me as i don't use docker for services thats needed here